swayfmt/CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Contributing to the Sway Language Formatter
+
+Firstly, thank you for taking interest in advancing the Sway language formatter! This guide will walk you through installation and best practices for contributing to this project.
+
+## Installation
+
+In order to see `swayfmt` in action you will need both `swayfmt` and the [`forc-fmt`](../forc-plugins/forc-fmt/) plugin. As previously stated, you can install these via [`fuelup`](https://github.com/FuelLabs/fuelup) which will install the complete Fuel toolchain, or follow the instructions below to install from source. If you've previously installed `fuelup`, you'll need to remove the binary before installing from source.
+
+> **Note:** `cargo` is a prerequisite to this build.
+
+```sh
+# 1. move to your preferred directory
+cd ~/Code/
+# 2. clone the Sway repo
+git clone https://github.com/FuelLabs/sway.git
+# 3. build from manifest and move the compiled result to your `.cargo/bin` folder
+cargo build --manifest-path ~/sway/forc-plugins/forc-fmt/Cargo.toml mv ~/sway/target/debug/forc-fmt ~/.cargo/bin
+```
+
+### Testing
+
+Move to the `src` folder of your project, and execute the binary:
```

Comment:
Does it have to be in the `src`? Pretty sure the current formatter allows you to run it from outside of `src` too.

Reply:
Ah, right you are, as long as there is a `Forc.toml` it should work